### PR TITLE
Added text to describe extension conformance in separate files and ad…

### PR DESCRIPTION
**Pull request by NSLeoFont**
# About
This PR adds description of how to create protocol extensions for objects conforming to them

# How

# Related Tasks
https://digitalcrew.teamwork.com/#/tasks/12651325

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -260,6 +260,15 @@ class MyViewController: UIViewController, UITableViewDataSource, UIScrollViewDel
...
 ```
 
+When putting protocol conformance for an object in an extension as previously mentioned, consider creating a folder and separated files to put its ancillary extensions therefore reducing the file of the initial component and facilitating looking for specific conformance.
+
+The container folder should have the name of the component (E.g., MyViewController) and inside the folder the main component should be placed alongside with the extensions, which will be named using the following convention: `ComponentName+ConformanceName`.
+
+**Preferred:**
+
+![Protocol Conformance Extensions](screens/protocolConformanceExtensions.png)
+
```
Comment (dfreniche) [inline README.markdown]: @NSLeoFont I'll also add an ASCII art folder structure, or better in place of a GIF. Leaner and cleaner. Here you have it:

```
[MyViewController] (folder)
|- MyViewController.swift
|- MyViewController+SomeDelegate.swift
|- MyViewController+SomeViewProtocol.swift
|- MyViewController+UITableViewDelegate.swift
|- MyViewController+Themable.swift
|- MyViewController+StoryboardInstantiable.swift
|- MyViewController+UITextViewDelegate.swift


```

**Review by dfreniche — CHANGES_REQUESTED** [review]

**Review by rkreutz-teamwork — APPROVED** [review]
Other than what Diego pointed out, LGTM

File: README.markdown (hunk 1 above)
Comment (NSLeoFont) [reply README.markdown]: Long life to the 80s! Viva ascii-art! Done in 7250d35473cbe05098ee6c909d0c687bba966e3f

**Review by dfreniche — APPROVED** [review]
Looks Quite ASCII To Me

**Review by josealobato — APPROVED** [review]

**Comment by dfreniche** [comment]
Changed branch to abandon gitflow and only use `master` per: https://digitalcrew.teamwork.com/#/tasks/12739917?c=5855155&

**Pull request merged**